torch_glow/src/PyTorchFileLoader.h

```diff
@@ -26,10 +27,20 @@ namespace glow {
 class PyTorchFileLoader {
 public:
   /// Takes a model file \p fileName, loads model into torch Module \p module,
-  /// reports error assigning \p errPtr.
-  PyTorchFileLoader(const std::string &fileName,
-                    std::shared_ptr<torch::jit::script::Module> &module,
-                    llvm::Error *errPtr);
+  /// \returns error if any.
+  static llvm::Error
+  loadPyTorchModel(const std::string &fileName,
+                   std::shared_ptr<torch::jit::script::Module> &module);
+
+  /// Takes a model file \p fileName, loads optimized graph into Glow Function
+  /// \p F and input \p inputPlaceholders, output \p outputPlaceholders
+  /// placeholders, \returns error if any.
```

Comment:
Can you add some comments about the thread_local stuff here. How should this be used to take advantage of that? For example, "call this once per inference thread for a model" or something like that